### PR TITLE
[develop][FIX][i5439]add certificate number1 url

### PR DESCRIPTION
**Pull request by ghost**
@Elico-Corp/odoo-reviewers 
please review

### CODE REVIEW

File: website_certificate/__openerp__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'Website Certificate',
-    'version': '8.0.0.0.3',
+    'version': '8.0.0.1.3',
```
Comment (elicoidal) [inline website_certificate/__openerp__.py]: 8.0.1.0.0

File: website_certificate/__openerp__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
     'category': 'website',
     'depends': ['base', 'website'],
     'author': 'Elico-Corp,Odoo Community Association (OCA)',
```
Comment (elicoidal) [inline website_certificate/__openerp__.py]: remove OCA

**Review by elicoidal — APPROVED** [review]
LGTM. needs testing and deploy

File: website_certificate/certificate_view.xml [hunk 3]
```diff
@@ -44,7 +48,9 @@
                         <span t-field="website.certificate_logo1"
                               t-field-options='{"widget": "image", "style": "display: inline; vertical-align: baseline;"}'/>
                     </a>
-                    <span t-field="website.certificate_number1"/>
+                    <a t-attf-href="#{website.certificate_number1_url}" target="_BLANK">
```
Comment (seb-elico) [inline website_certificate/certificate_view.xml]: s/_BLANK/_blank/

**Review by seb-elico — CHANGES_REQUESTED** [review]
Great job!

**Comment by ghost** [comment]
@Elico-Corp/odoo-reviewers modify the version author and capitalization, please review

**Review by seb-elico — APPROVED** [review]
LGTM

**Pull request merged**